tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -213,6 +387,7 @@ public boolean onKeyDown(final int keyCode, final KeyEvent event) {
     return super.onKeyDown(keyCode, event);
   }
 
+  protected abstract void processImageRGBbytes(byte[] luminance) ;
```

Comment:
it was supposed to be used with Detector demo, because it requires the luminance to draw the squares, TBH I left that thing mistake